modules/arm_plugin/src/transformations/quantize_fusion.cpp

```diff
@@ -349,6 +352,10 @@ ArmPlugin::pass::DequantizeInputFusion::DequantizeInputFusion() {
             if (output->output(0).get_target_inputs().size() != 1 || !realType.is_real()) {
                 return false;
             }
+            auto node = output->output(0).get_target_inputs().begin()->get_node()->shared_from_this();
+            if (ngraph::is_type<opset::AvgPool>(node) && node->get_input_shape(0).size() == 5) {
```

Comment:
It would be better to move this check inside `if` at line 390/395.
This check ensure QInfo will be configured for AvgPool with INT8 output, while AvgPool with F32 output will be processed by other branch.
So for the sequence U8->Mul-(F32)->AvgPool->F32 transformation will yield
U8->Dequantize-(f32)->AvgPool->F32
instead of U8->Dequantize-(F32)->Mul-(F32)->AvgPool->F32 prepared by `DequantizeOnInputs` transformation that follow